dashboard/pkg/epinio/index.ts

```diff
@@ -41,4 +50,47 @@ export default function(plugin: IPlugin) {
 
   // Add hooks to Vue navigation world
   plugin.addNavHooks(onEnter, onLeave);
+
+  // Add action button in the menu of each object belonging to Epinio's applications
+  plugin.addAction(
+    ActionLocation.TABLE,
+    {
+      resource: [
+        // put other epinio application's objects
+        'apps.deployment',
```

Comment:
we can only link from a deployment, pod, or a workload if it is a deployment or pod.

we might be able to do service as well, which goes to a service instance

Reply:
'batch.job' can be removed as well